stl/inc/atomic

```diff
@@ -737,7 +737,7 @@ struct _Atomic_storage<_Ty, 8> { // lock-free using 8-byte intrinsics
 
         return _Temp;
     }
-#else // ^^^ _M_IX86 / !_M_IX86 vvv
+#else // ^^^ _M_IX86 on clang / !_M_IX86 on clang vvv

```

Comment:
This comment is now ambiguous. It intends to say "not (x86 on clang)" but can be read as "(not x86) on clang".

I recommend using [De Morgan's laws](https://en.wikipedia.org/wiki/De_Morgan%27s_laws) for this comment (and the second copy), which would make it verbose, but precise:

```suggestion
#else // ^^^ defined(_M_IX86) && defined(__clang__) / !defined(_M_IX86) || !defined(__clang__) vvv
```